extension/test/test-logger.js

```diff
@@ -10,7 +10,13 @@ var verifyAsyncOutput = function (test, logger, testFunc) {
   var message = '';
 
   return function (status) {
-    var line = {}, hasMore;
+    var line = {}, hasMore, isSuccessStatus;
+
+    isSuccessStatus = typeof status == "number";
+    test.assert(isSuccessStatus, status);
+    if (!isSuccessStatus) {
+      return;
+    }
```

Comment:
> [...] If that is the case it should be enough to use `assert.ok(typeof status == "number")`.

Yes, I'd like to return early, because the file write failed.
I'll try that, thanks!